README.md

```diff
@@ -1,11 +1,11 @@
 # Open edX Brand Package Interface
 
-**Warning** This package will no longer recieve updates and the latest version
+**Warning** This package will no longer receive updates. The latest version
 of this package can be found at
 https://www.npmjs.com/package/@openedx/brand-openedx
 
 This project contains the default branding assets and style used in Open edX
-applications. It is published on npm as `@edx/brand-openedx`.
+applications. It is published on NPM as `@edx/brand-openedx`.
```

Comment:
```suggestion
applications. It is published on npm as `@edx/brand-openedx`.
```

`npm` is officially all lowercase https://en.wikipedia.org/wiki/Npm,_Inc.

I'm not sure how we generally handle situations like this.